src/user-ratings/UserRatings.js

```diff
@@ -1,21 +1,21 @@
 import React, { Component } from 'react';
 import "./UserRatings.css"
 import SingleMovie from '../single-movie/SingleMovie.js'
-import { getMovieRatings } from '../apiCalls.js'
+import { getMovieRatings, deleteMovieRatings } from '../apiCalls.js'
 
 
 export default class UserRatings extends Component {
   constructor(props) {
     super(props);
     this.state = {
-      selectedRating: null,
-      error: ''
+      selectedRating: 1,
```

Comment:
I would not have thought of just putting 1 here to solve our rating issue. Great catch!